service/app/util/RequestInterceptor.java

```diff
@@ -139,6 +139,13 @@ public static String verifyRequestData(Http.Context ctx) {
     }
   }
 
+  private static boolean isRequestPrivate(String path) {
+    if (path.contains(JsonKey.PRIVATE)) {
```

Comment:
1  - Return the result of conditional directly - no need to return boolean, explicitly.
2. contains ("/" + private + "/") is desirable. Because if there is a semi-word in path like privateFields